util/logger.go

```diff
@@ -25,12 +35,17 @@ func LoggerInit(verbose bool) {
 	if verbose {
 		verboseWriter = os.Stdout
 	}
+
+	s, _ := spun.NewSpinner(spun.Dots)
 	logger = &RigLogger{
 		Info:      log.New(os.Stdout, color.BlueString("[INFO] "), 0),
 		Warning:   log.New(os.Stdout, color.YellowString("[WARN] "), 0),
 		Error:     log.New(os.Stderr, color.RedString("[ERROR] "), 0),
 		Verbose:   log.New(verboseWriter, "[VERBOSE] ", 0),
+		Message:   log.New(os.Stdout, " - ", 0),
```

Comment:
This doesn't appear to be used

Reply:
Sorry, i see where it is used. What is the idea behind Message?

Reply:
It will be pulled if continuing refactor doesn't find a use for it. I wanted to stake some concepts aside from log severity for the channels.